### DISCUSSION TITLE
Track commit hash of Jenkins libraries

### DISCUSSION THREAD

**Discussion by jjcaballero** [renovatebot/renovate#13304] [Request Help]
I am looking at the possibility of updating the commit hash that defines a Jenkins library. In practice, this would be to track a _Jenkinsfile_ with a content something like:
```
@Library([ 'library-name@225b115dab5f68d292b0d1ad9020aedce8cf893f']) _
...
```
I am thinking of the following combination:
- Versioning: git
- Datasource: git-refs
- Manager: regex

Do you see any blocking issue?

I was thinking of gitsubmodule manager but I guess this can't work because the repository url wouldn't be defined at all. That's why I am proposing "git-refs" as datasource.

Thanks!

**Comment by viceice** [comment]
Regex manager should work
